src/sb_pointcloud_processing/scripts/nodelet_launch.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
We should add a descriptive comment here explaining what this file is doing and why it exists.